packages/api/tests/lib/test-granule-delete.js

```diff
@@ -260,7 +260,7 @@ test.serial('deleteGranuleAndFiles() succeeds if a file is not present in S3', a
     cumulus_id: pgGranule.cumulus_id,
   });
 
-  await deleteGranuleAndFiles({
+  const details = await deleteGranuleAndFiles({
```

Comment:
This might be a mistake? It looks like we don't do anything with `details`.

Reply:
Mistake addressed.